.github/workflows/deploy.yml

```diff
@@ -14,32 +14,33 @@ jobs:
       - name: 🎛 Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version: 16.16
-          cache: 'npm'
+          node-version: 18.12.1
+          cache: 'yarn'
 
       - name: ⚙️ Install
-        run: npm ci
+        run: yarn
```

Comment:
This is the build script that executes on the main branch after the PR is closed.